source/learners/sfd_discussions/getting_started.rst

```diff
@@ -0,0 +1,91 @@
+.. _Getting started with Discussions:
+
+##################################
+Getting started with Discussions
+##################################
+
+This section describes where to find discussions and the structure and elements of course discussions.
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+.. _Where to Find Discussions:
+
+**************************
+Where to find Discussions
+**************************
+
+You can access course discussions in two ways:
+
+1. From the course navigation
+
+   Click the **Discussions** tab at the top of your course. This opens the full discussion page, 
+   where you can browse all topics, search, and post new questions or discussions.
+
+   .. figure:: /_images/learners/Discussions_course_nav.png
+      :alt: The Discussions page can be accessed from the course navigation bar.
+      :width: 600px
+      :align: center
+
+      The Discussions page can be accessed from the course navigation bar.
+   
+
+2. From within a course unit
+
+   Some course units include a discussion panel on the right side of the page.  
+   You can use this panel to read or take part in discussions related specifically to that unit.
+
```

Comment:
I would move the Discussions_sidebar.png to here, before the explanatory text

Reply:
Done